entrypoint.sh

```diff
@@ -3,4 +3,8 @@
 set -euo pipefail
 
 ./prepare.sh
-/usr/bin/bootc-image-builder --store /store --rpmmd /rpmmd --output /output  "$@"
+tmpout="$(mktemp -p / -d)"
+/usr/bin/bootc-image-builder --store /store --rpmmd /rpmmd --output "${tmpout}" "$@"
+echo "Moving artifacts to output directory."
+cp -rf "${tmpout}/." /output
```

Comment:
I don't quite understand why we have this bash script instead of doing things in Go?

Also in the end it'd be a good bit more efficient to create images in their final destination to avoid copying.

Reply:
We have the script so we can call `/prepare.sh` which we need for other reasons.  I could do it in Go, that's not an issue.

But yes, this essentially means we're copying the artifact twice.  It's not terrible, but it'd be simpler to export directly to our destination.  The problem there is what I mentioned in the original comment: `cp -a` will fail to preserve the ownership if it can't set it on the destination, which is what's happening here and osbuild using `cp -a` to export artifacts is important for all sorts of scenarios (copying entire system root trees for example).